btservice/btservice.cpp

```diff
@@ -37,7 +37,35 @@ btservice::btservice(openauto::configuration::IConfiguration::Pointer config)
     {
         OPENAUTO_LOG(info) << "[btservice] Service registered, port: " << cServicePortNumber;
     }
+    connectToBluetooth(QBluetoothAddress(QString::fromStdString(config->getLastBluetoothPair())), address);
 }
 
+void btservice::connectToBluetooth(QBluetoothAddress addr, QBluetoothAddress controller)
```

Comment:
i think adding the config option to toggle this is a good idea

Reply:
Yup, that'll probably be my next (and final) update to this branch